core/templates/dev/head/services/ConstructTranslationIdsService.js

```diff
@@ -16,14 +16,16 @@
  * @fileoverview Service to dynamically construct translation ids for i18n.
  */
 
-oppia.factory('ConstructTranslationIdsService', function() {
-  return {
+oppia.factory('ConstructTranslationIdsService', [
+  function() {
+    return {
     // Construct a translation id for library from name and a prefix.
```

Comment:
Indent by 2 along with the the next line.

Reply:
Done.